common/src/main/kotlin/no/nav/su/se/bakover/common/Config.kt

```diff
@@ -19,6 +19,9 @@ object Config {
     val azureClientId = env["AZURE_CLIENT_ID"] ?: "24ea4acb-547e-45de-a6d3-474bd8bed46e"
     val azureBackendCallbackUrl = env["BACKEND_CALLBACK_URL"] ?: "http://localhost:8080/callback"
     val azureRequiredGroup = env["AZURE_REQUIRED_GROUP"] ?: "requiredgroup"
+    val azureGroupAttestant = env["AZURE_GROUP_ATTESTANT"] ?: ""
```

Comment:
Har dere lagt inn de aktuelle gruppene for hvert miljø i Vault?

Eventuelt kunne vi undersøkt om det er "trygt" at disse verdiene er default i koden. Dette vil isåfall kreve at vi håndterer switch mellom dev/prod.

Reply:
Dette høres ut som det blir vanskeligere å kjøre appen lokalt og vanskeligere å "bytte" mellom rollene. Det hadde vært fint å fått en enkel løsning på det nå + lagt inn guide i readme.